app/src/main/java/com/infomaniak/mail/data/models/thread/Thread.kt

```diff
@@ -65,16 +65,15 @@ class Thread : RealmObject {
     var from: RealmList<Recipient> = realmListOf()
     var to: RealmList<Recipient> = realmListOf()
     var subject: String? = null
-    var size: Int = 0
     @SerialName("has_attachments")
     var hasAttachments: Boolean = false
     var hasDrafts: Boolean = false
     @SerialName("flagged")
     var isFavorite: Boolean = false
+    @SerialName("answered")
```

Comment:
This is useless, we create our own threads, we don't need to deserialise them _(except for the Search, but the only variable we use is the `messages` one)_.
All `@SerialName` in this file should be removed.